src/Nethermind/Nethermind.Serialization.Json/BigIntegerConverter.cs

```diff
@@ -67,7 +67,7 @@ public override BigInteger ReadJson(JsonReader reader, Type objectType, BigInteg
                 return (long) reader.Value;
             }
 
-            string s = (string) reader.Value;
+            string s = reader.Value?.ToString();
```

Comment:
can you add numbers with this change?
the CLI is a javascript engine and it can worh number - java adds strings vi concatenation